web/src/app/lists/ControlledPaginatedList.tsx

```diff
@@ -57,6 +55,9 @@ export interface ControlledPaginatedListProps
   searchAdornment?: ReactElement
 
   items: CheckboxItemsProps[] | PaginatedListItemProps[]
+
+  CreateDialog?: ComponentType<{ onClose: () => void }>
```

Comment:
This should have the suffix `Component` since it's expected not to have been rendered yet.